src/containers/ViewerPanel/index.tsx

```diff
@@ -261,13 +264,25 @@ class ViewerPanel extends React.Component<ViewerPanelProps, ViewerPanelState> {
         setIsLooping(!isLooping);
     }
 
+    public handleIncomingConvertedFile(data: TrajectoryFileInfo) {
+        const { receiveConvertedFile, conversionProcessingData } = this.props;
+        const fileId = conversionProcessingData.fileId + ".simularium";
+        const hasTitle =
+            data.trajectoryTitle !== undefined &&
+            data.trajectoryTitle.length > 0;
+        const title = hasTitle ? data.trajectoryTitle : fileId;
```

Comment:
are there cases where 
```suggestion
        const title = data.trajectoryTitle  || fileId;
```
didn't work? just curious about your check for both undefined and the length